docs/technical_reference/property_models/ASM2D_GHG.rst

```diff
@@ -0,0 +1,321 @@
+  ":lime:`Nitrite`", ":math:`S_{NO2}`", "S_NO2"
+  ":blue:`Nitrate`", ":math:`S_{NO3}`", "S_NO3"
+  "Dinitrogen", ":math:`S_{N2}`", "S_N2"
+  "Inorganic soluble phosphorus, primarily ortho-phosphates", ":math:`S_{PO4}`", "S_PO4"
+  "Inorganic carbon", ":math:`S_{IC}`", "S_IC"
+  "Inert particulate organic material", ":math:`X_I`", "X_I"
+  "Slowly biodegradable substrates", ":math:`X_S`", "X_S"
+  "Heterotrophic organisms", ":math:`X_H`", "X_H"
+  "Phosphate-accumulating organisms", ":math:`X_{PAO}`", "X_PAO"
+  "Poly-phosphate", ":math:`X_{PP}`", "X_PP"
+  "A cell internal storage product of phosphorus-accumulating organisms, primarily comprising poly-hydroxy-alkanoates (PHA)", ":math:`X_{PHA}`", "X_PHA"
+  ":red:`Autotrophic nitrifying organisms`", ":math:`X_{AUT}`", "X_AUT"
+  ":lime:`Ammonia oxidizing bacteria`", ":math:`X_{AOB}`", "X_AOB"
+  ":lime:`Nitrite oxidizing bacteria`", ":math:`X_{NOB}`", "X_NOB"
+  "Potassium", ":math:`S_{K}`", "S_K"
+  "Magnesium", ":math:`S_{Mg}`", "S_Mg"
+
+State variables
+---------------
+:red:`Red` text indicates the state variable has been removed in the Modified ASM2d model.
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Total volumetric flowrate", ":math:`Q`", "flow_vol", "None", ":math:`\text{m}^3\text{/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
+   "Component mass concentrations", ":math:`C_j`", "conc_mass_comp", "[j]", ":math:`\text{kg/}\text{m}^3`"
+
+Stoichiometric Coefficients
+---------------------------
+:red:`Red` text indicates the stoichiometric coefficient has been removed in the Modified ASM2d model, :lime:`lime` text indicates the stoichiometric coefficient has been added, and :blue:`blue` text indicates the coefficient has had its value changed from its base ASM2d implementation.
+
+:Note: Parameter values found in asm2dinit_bsm2.c: https://app.box.com/file/1812331425408
+
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter", "Default Value", "Units"
+
+   "C content of inert soluble COD S_I", ":math:`i_{CSI}`", "i_CSI", 0.36718, ":math:`\text{dimensionless}`"
+   "C content of inert soluble COD S_F", ":math:`i_{CSF}`", "i_CSF", 0.31843, ":math:`\text{dimensionless}`"
+   "C content of inert soluble COD S_A", ":math:`i_{CSA}`", "i_CSA", 0.375, ":math:`\text{dimensionless}`"
+   "C content of inert soluble COD X_I", ":math:`i_{CXI}`", "i_CXI", 0.36178, ":math:`\text{dimensionless}`"
+   "C content of inert soluble COD X_S", ":math:`i_{CXS}`", "i_CXS", 0.31843, ":math:`\text{dimensionless}`"
+   "C content of inert soluble COD X_B", ":math:`i_{CXB}`", "i_CXB", 0.36612, ":math:`\text{dimensionless}`"
+   ":lime:`C content of inert soluble COD X_PHA`", ":math:`i_{CXPHA}`", "i_CXPHA", 0.3, ":math:`\text{dimensionless}`"
+   "N content of inert soluble COD S_I", ":math:`i_{NSI}`", "i_NSI", 0.06003, ":math:`\text{dimensionless}`"
```

Comment:
double-check descriptions and ensure they accurately align with source. I suppose the tricky part has to do with catching potential typo errors in the source, if any.